ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py

```diff
@@ -229,3 +229,37 @@ def test_linear(
         weight=weight,
         bias=bias,
     )
+
+
+def test_bilinear(
+    *,
+    dtype_x_weight_bias,
+    on_device,
+    fn_tree,
+    backend_fw,
+    frontend,
+    test_flags,
+):
+    dtype, x1, x2, weight, bias = dtype_x_weight_bias
```

Comment:
Please fetch the support dtypes for this test instead of reusing the earlier one for linear!

Reply:
Can you elaborate further?

Reply:
> Please fetch the support dtypes for this test instead of reusing the earlier one for linear using the `handle_frontend_test` decorator similar to other tests to configure the test correctly!

do you mean this?
@handle_frontend_test(
    fn_tree="paddle.nn.functional.common.bilinear",
    dtype_x_weight_bias=x_and_linear(
        dtypes=helpers.get_dtypes("valid", full=False),
    ),
)